django/test/html.py

```diff
@@ -96,6 +97,17 @@ def _count(self, element, count=True):
                     elif element in child:
                         return 1
             else:
+                if isinstance(element, RootElement) and element.children:
```

Comment:
I think you should add an explanation what this does or at least add a reference to the ticket to help future readers understand what this branch achieves.